src/include/platform/CHIPDeviceEvent.h

```diff
@@ -287,6 +287,13 @@ enum ActivityChange
 enum OtaState
 {
     kOtaSpaceAvailable = 0,
+    kOtaDownloadInProgress,
+    kOtaDownloadComplete,
+    kOtaDownloadFailed,
+    kOtaDownloadAborted,
+    kOtaApplyInProgress,
+    kOtaApplyComplete,
+    kOtaApplyFailed,
```

Comment:
A one-liner description in front of each of these will help

Reply:
Added the description.